docs/sdks/ios/deeplink.md

```diff
@@ -0,0 +1,43 @@
+---
+id: Universal Link
+title: Universal Link
+---
+
+## CleverPush Universal Link
+
+1. Select the Project.
+2. Select the Project target.
+3. Select the info tab.
+4. Select the url type.
+5. Select the url scheme and enter the url scheme (Example :- cleverpush)
+6. Go to the button of any view controller from where we need to share the url to another user 
+
+```swift
+@IBAction func btnHandlerShareURL(_ sender: Any) {
+
+        // From here user can share the cleverpush url to another user using the share activity
+        let path = "cleverpush://"
```

Comment:
is path always with the cleverpush:// scheme or does the customer have to specify his own scheme (if yes please clarify in the comment above)

Reply:
I have added comments with example also so customers can easily understand.